### ISSUE TITLE
REQ/Bug: Alt tabbing when a window is fullscreen should open windows above it.

### ISSUE THREAD

**Issue by Quackdoc** [pop-os/cosmic-comp#921]
Currently when an app is fullscreen, alt+tab will bring a new app into focus, but it is below the fullscreen app, inputs are properly sent to the app in question. 

The fullscreen app should remain "fullscreen" but apps and panels should be drawn above it. This is how plasma handles it, I'm not sure how gnome does.

Partially related to https://github.com/pop-os/cosmic-comp/issues/696

**Comment by vadim-su** [comment]
Yeah, it would be great. I want to switch windows when current window in Fullscreen mode. @Quackdoc have you ever try `Hyprland`? It allows swap Fullscreen windows without problems because it's just a modifier for the current window

**Comment by vadim-su** [comment]
I think it should work as well as it does for other DE managers. It works fine in hyprland and kde, so we need to implement similar behavior in cosmic-comp.

**Comment by vadim-su** [comment]
Maybe I can spend time at the weekends to find out how cosmic-comp works under the hood

**Comment by Daktyl198** [comment]
I'm having this same issue with Path of Exile 2, except that in my case it seems that it doesn't even change focus to the window below it. Trying to alt-tab to any window on the same monitor as the fullscreen application just re-focuses the fullscreen application.

**Comment by atiq-cs** [comment]
I am able to repro this bug: Application switching (Press Alt + Tab etc. ) doesn't work once in full screen mode for live streaming players (for example, the player on https://www.bloomberg.com/live or on youtube i.e, https://www.youtube.com/watch?v=dmW51BebI8c)

In more detail, this is the screenshot from bloomberg live player on full screen.

![Image](https://github.com/user-attachments/assets/27f7a937-b69c-441a-8ef6-cbfac9704afe)

Now, no keyboard shortcut will have any response until I exit the full screen! For example, pressing Super Key to bring the applications menu or Super + slash to show launcher, nothing works.

**Comment by Quackdoc** [comment]
@atiq-cs this issue is likely to be fixed by https://github.com/pop-os/cosmic-comp/pull/1481 If you wanted to try it

**Comment by atiq-cs** [comment]
@Quackdoc PR #1481 is still a draft, far away to be reviewed/merged, not sure what did you mean by trying it.

**Comment by git-f0x** [comment]
You can clone the branch and compile it. But yes, this will be fixed when that is merged.

**Comment by Quackdoc** [comment]
fix
